src/Components/ShopInfo/product-grid.jsx

```
@@ -0,0 +1,51 @@
+export default function ProductGrid({ productList }) {
+  const { userId, shopId } = useParams();
+
+  return (
+    <div className="flex justify-start gap-8">
+      {productList.map((product) => {
+        const authedRoute = `/user-dashboard/${userId}/shop-info/${shopId}/product-info/${product.id}`;
+        const unauthedRoute = `/shop-info/${shopId}/product-info/${product.id}`;
+        console.log(userId ? authedRoute : unauthedRoute);
+        
+        return (
+          <Link to={userId ? authedRoute : unauthedRoute}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)